source/Sashimi.Aws.Tests/CloudFormation/CloudFormationActionHandlerFixture.cs

```diff
@@ -0,0 +1,319 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using NuGet.Packaging;
+using NuGet.Versioning;
+using NUnit.Framework;
+using Sashimi.Aws.ActionHandler;
+using Sashimi.Tests.Shared;
+using Sashimi.Tests.Shared.Server;
+
+namespace Sashimi.Aws.Tests.CloudFormation
+{
+    public class CloudFormationActionHandlerFixture
+    {
+        const string AwsStackRole = "arn:aws:iam::968802670493:role/e2e_buckets";
+        // BucketName must use the following prefix, otherwise the above stack role will not have permission to access 
+        const string ValidBucketNamePrefix = "cfe2e-";
```

Comment:
The bucket prefix has changed from `e2e-test` - I vaguely remember that this might be important. @matt-richardson do you remember?

Reply:
I have no recollection... Either that, or I have deliberately blocked it from my :brain: